src/main/java/com/metsci/laproc/tools/PointInfoPanel.java

```diff
@@ -29,30 +29,37 @@ public PointInfoPanel() {
     }
 
     /**
-     * updates the rows of the point info panel
+     * updates the rows of the point info panel for each displayed graph
      */
     public void update(GraphPoint[] points){
-        Map<String, Double> data = points[0].getAnalytics();
+        this.panel.remove(pane);
+        pane = new JScrollPane();
```

Comment:
If you wanted to avoid creating a whole new scroll pane, I think there is a removeAll function that removes all components to the frame and you could then just add all the new panels.